stressor/stress-test.mts

```diff
@@ -52,7 +54,17 @@ const program = new Command()
   .option('-b, --branch <string>', 'Git branch', 'main')
   .option(
     '-f, --results-from-file',
-    "Load results from local 'allResults.json' file"
+    'Load results from json file instead of live collection.'
+  )
+  .option(
+    '-j, --json-output <file>',
+    'Write/read results collection data to json file.',
+    'stress-run.json'
```

Comment:
[The project's `.gitignore` file still references `stressor-run.json`.](https://github.com/bocoup/aria-at-gh-actions-helper/blob/89dcc718d07958ca9b038ccaa50e26e0923d531e/.gitignore#L2) It might be best to keep that name for consistency.